internal/backends/error.go

```
@@ -40,6 +40,11 @@ const (
 	ErrorCodeCollectionAlreadyExists
 
 	ErrorCodeInsertDuplicateID
+
+	ErrorCodeIndexNameIsEmpty
+	ErrorCodeIndexAlreadyExists
+	ErrorCodeIndexOptionsConflict
+	ErrorCodeIndexKeySpecsConflict
```

All those errors are never handled and therefore should not be introduced